docs/prometheus-exporter.md

```diff
@@ -0,0 +1,24 @@
+# Prometheus exporter
+
+This document describes how to configure `concordium-node` to run a [Prometheus exporter](https://prometheus.io/) and documentation for what each of the exported metrics mean.
+
+## Enable the exporter
+
+Concordium node does not run the Prometheus exporter by default. To enable it at node startup, the listening port for the exporter must be provided either through a command line argument `--prometheus-listen-port` or the environment variable `CONCORDIUM_NODE_PROMETHEUS_LISTEN_PORT`.
+
+Likewise, the IP address to listen on can be specified using command line argument `--prometheus-listen-addr` or environment variable `CONCORDIUM_NODE_PROMETHEUS_LISTEN_ADDRESS`. Defaults to `127.0.0.1` when not provided.
+
+To verify the exporter is running open the provided listen address and port in a browser. Which should display the text "Operational".
+
+## Push metrics to a Pushgateway
```

Comment:
We should probably mention also that you can't do both at the same time, push and serve. If I remember correctly.

Reply:
I don't see that limitation in our own code or documented in the prometheus library

Reply:
You're right, it's just the logging that is broken.
```rust
    if conf.prometheus.prometheus_listen_port.is_some() {
        info!("Enabling prometheus server");
    } else if let Some(ref push_gateway) = conf.prometheus.prometheus_push_gateway {
        info!("Enabling prometheus push gateway at {}", push_gateway);
    };
```

Reply:
I can make a PR fixing that

Reply:
Please do.